docs/features/autoupdater.rst

```diff
@@ -35,9 +35,9 @@ A fully automated nightly build could use the following commands:
 ::
 
     git pull
-    (cd site && git pull)
+    (git -C site pull)
     make update
-    make clean
+    make clean GLUON_TARGET=ar71xx-generic
```

Comment:
why do you have to add a target here?

Reply:
`make clean` doesn't work for me, with GLUON_TARGET only 

It is mention in the Getting Started guide, too
https://gluon.readthedocs.io/en/v2018.1.x/user/getting_started.html#cleaning-the-build-tree